include/open62541/plugin/log.h

```diff
@@ -52,103 +52,109 @@ typedef struct {
      * according to the rules of the printf command. Use the convenience macros
      * below that take the minimum log level defined in ua_config.h into
      * account. */
-    void (*log)(void *logContext, UA_LogLevel level, UA_LogCategory category,
+    void (*log)(void *logContext, UA_LogLevel level, UA_LogCategory category, unsigned int line,
                 const char *msg, va_list args);
 
     void *context; /* Logger state */
 
     void (*clear)(void *context); /* Clean up the logger plugin */
 } UA_Logger;
 
-static UA_INLINE UA_FORMAT(3,4) void
-UA_LOG_TRACE(const UA_Logger *logger, UA_LogCategory category, const char *msg, ...) {
+static UA_INLINE UA_FORMAT(4,5) void
+UA_LOG_TRACE2(const UA_Logger *logger, UA_LogCategory category, unsigned int line, const char *msg, ...) {
```

Comment:
When adding an argument here, you should probably also add `(void) line;` in case UA_LOGLEVEL > 100. Ditto for the functions below.